sdk/entra/WebJobs.Extensions.AuthenticationEvents/src/WebJobs.Extensions.AuthenticationEvents/Templates/ActionableTemplate.json

```diff
@@ -0,0 +1,6 @@
+﻿{
```

Comment:
Are these needed by tests or for the local development experience?

Reply:
These are needed by the framework to create the outputs.